src/Forge.php

```diff
@@ -135,16 +135,19 @@ public function create()
     /**
      * Returns single server.
      *
-     * @param int $serverId
+     * @param int     $serverId
+     * @param boolean $reload   (optional) indicates whether the server should be reloaded
      *
      * @return \Laravel\Forge\Server
      */
-    public function get(int $serverId)
+    public function get(int $serverId, bool $reload = null)
```

Comment:
Default value for booleans should be either `true` or `false` (`false` in your case), but not `null`. Nulls are better suited for optional strings or objects (it depends, of course).